### DISCUSSION TITLE
How to integrate Tabler to a Ruby on Rails app

### DISCUSSION THREAD

**Discussion by jkvithanage** [tabler/tabler#1435] [General]
Does anyone know how to integrate tabler into a rails app? 

I tried,
1. yarn add @tabler/core
2. Add @import '@tabler/core/src/scss/tabler' ;  to app/assets/stylesheets/application.scss

But didn't work.

**Comment by Smallz2** [comment]
Try adding 
`@import "@tabler/core/dist/css/tabler";`

**Reply by jkvithanage** [reply]
Thanks, it worked!
